src/objects/Term_Grade__c.object

```diff
@@ -0,0 +1,233 @@
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>CancelEdit</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>Clone</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>Delete</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>Edit</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>List</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>New</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>SaveEdit</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>Tab</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>View</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <allowInChatterGroups>false</allowInChatterGroups>
+    <compactLayoutAssignment>SYSTEM</compactLayoutAssignment>
+    <deploymentStatus>Deployed</deploymentStatus>
+    <description>Stores student grades for a specific grading period within a Term.</description>
+    <enableActivities>true</enableActivities>
+    <enableBulkApi>true</enableBulkApi>
+    <enableFeeds>false</enableFeeds>
+    <enableHistory>false</enableHistory>
+    <enableLicensing>false</enableLicensing>
+    <enableReports>true</enableReports>
+    <enableSearch>true</enableSearch>
+    <enableSharing>true</enableSharing>
+    <enableStreamingApi>true</enableStreamingApi>
+    <fields>
+        <fullName>Attempted_Credits__c</fullName>
+        <description>The number of credits attempted for this course and grading period.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The number of credits attempted for this course and grading period.</inlineHelpText>
+        <label>Attempted Credits</label>
+        <precision>18</precision>
+        <required>false</required>
+        <scale>0</scale>
+        <trackTrending>false</trackTrending>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Course_Connection__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The class in which the student is received this grade.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The class in which the student is received this grade.</inlineHelpText>
+        <label>Course Connection</label>
+        <referenceTo>Course_Enrollment__c</referenceTo>
+        <relationshipLabel>Term Grades</relationshipLabel>
+        <relationshipName>Term_Grades</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
```

Comment:
Should we add a new related list to the Course Connection layout? Or have that be a "pro tip" / part of the K12 kit? (I also asked this in the spec doc, I think)

Reply:
If we add it, can we remove it later? if the answer is no - let's not go that far yet. We can always add it in a subsequent release. My vote is for "Pro Tip" for now. @robertbwatson

Reply:
We can remove it, but it wouldn't be removed for the new customers that got the update. We should be consistent here though - we've added them for other objects, I think. Thoughts @lifewithryan?

Reply:
@lifewithryan  @xinnanlisf Did we decide today we're adding the Term Grades to the Course Connection by default (for new installs), or are we keeping it off?

Reply:
I am okay leaving it off for now since we can always add it in for new installs later, fyi